.idea/electron-builder.iml

```diff
@@ -5,6 +5,7 @@
       <excludeFolder url="file://$MODULE_DIR$/assets" />
       <excludeFolder url="file://$MODULE_DIR$/example-app" />
       <excludeFolder url="file://$MODULE_DIR$/node_modules" />
+      <excludeFolder url="file://$MODULE_DIR$/out" />
```

Comment:
In the out new file `electron-builder.d.ts` is located on publish. To avoid strange completion in the IDE in the test files, it is better for now just exclude `out` completely.